src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopy = { ...state };
+  const newArrayState = [];
```

Comment:
I'd give it another name
this array is meant contain history of state, its versions, etc.